aws/studio_hpo_container_setup.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>Remove these, most details about HPO can be moved to the hpo notebook, this will mainly be focused about the container build and what it includes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>